File: 프로그래머스/0/181912. 배열 만들기 5/배열 만들기 5.cs
using System;
using System.Collections.Generic;
public class Solution {
    public int[] solution(string[] intStrs, int k, int s, int l) {
        List<int> answer = new List<int>();
        for(int i = 0 ; i < intStrs.Length;i++)
        {
            string sub = intStrs[i].Substring(s,l);
            int num = int.Parse(sub);
            if(num > k)
                answer.Add(num);
        }
        return answer.ToArray();
    }
}

File: 프로그래머스/0/181912. 배열 만들기 5/배열 만들기 5.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<string> intStrs, int k, int s, int l) {
    vector<int> answer;
    for(int i = 0; i < intStrs.size(); i++)
    {
        string temp = intStrs[i].substr(s,l);
        int num = stoi(temp);
        if(num > k)
            answer.push_back(num);
    }
    return answer;
}